src/openrct2/game.c

```diff
@@ -1420,7 +1410,7 @@ void game_load_or_quit_no_save_prompt()
  */
 void game_init_all(sint32 mapSize)
 {
-    gInUpdateCode = true;
+    gInMapInitCode = true;
```

Comment:
You still end up checking both of these variables, so it would look like you can just leave it the way it was.